### PR TITLE
Fix the test append the correct paths

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: tests/aws/test_scylla_configure.py [hunk 1]
```diff
@@ -10,8 +11,12 @@
 from pathlib import Path
 from http.server import HTTPServer, BaseHTTPRequestHandler
 from socketserver import ThreadingMixIn
+
+sys.path.append('../../aws')
```
Comment (bentsi) [inline tests/aws/test_scylla_configure.py]: why did you need this?

Comment (fruch) [reply tests/aws/test_scylla_configure.py]: cause test won't work without it
try running from the command line and you'll see the issue:
```
python3.7 -m unittest test_scylla_configure.py
```

Comment (fruch) [reply tests/aws/test_scylla_configure.py]: pycharm add some of directories into the sys.path by default (like the root of the repo), the command line doesn't do that

Comment (bentsi) [reply tests/aws/test_scylla_configure.py]: I mean it is enough to add `sys.path.append('../..')`
